datahub-web-react/src/app/home/HomePageHeader.tsx

```diff
@@ -228,7 +265,14 @@ export const HomePageHeader = () => {
                     />
                     {searchResultsToShow && searchResultsToShow.length > 0 && (
                         <SuggestionsContainer>
-                            <SuggestedQueriesText strong>Try searching for</SuggestedQueriesText>
+                            <SuggestionsHeader>
+                                <SuggestedQueriesText strong>Try searching for</SuggestedQueriesText>
+                                <ExploreAllButtonContainer>
```

Comment:
is this `ExploreAllButtonContainer` necessary if you added `SuggestionsHeader` with `justify-content: space-between;`?

Reply:
oh nice... I think it should not be!